README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
Ok, that makes sense. Might be worthwhile adding this very explanation in there in case others ever wonder about it.